<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Donneur</class>
 <widget class="QDialog" name="Donneur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1788</width>
    <height>1048</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Donneur</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font: 75 10pt &quot;MV Boli&quot;;
}
#Donneur
{

background-color: QLinearGradient(x1: 0,y1: 0,x2: 0,y2: 1,stop: 0 #e66465, stop:1 #9198e5);
border-radius:15px;
}
QLabel
{
	color: rgb(0, 0, 0);
}
QPushButton
{
	
	border-radius:60px;
	background-color: rgb(255, 171, 173);
}
QPushButton
{
border-radius:15px;
}
QPushButton:hover
{
	color:white;
	background-color: rgb(0, 0, 0);
	border-radius :15px;
}
Qlabel{
height:50px;
width:40px;
}

QToolButton
{
background: transparent;
border-radius:60px;

}</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>1811</width>
     <height>881</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MV Boli</family>
     <pointsize>10</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>1241</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>431</width>
      <height>821</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
      <pointsize>10</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLineEdit" name="lineEdit_nom">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_adresse">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>280</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Veuillez saisir une adresse spécifique</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_mail">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>350</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Exemple : xyz@exemple.abc</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_annuler">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>490</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Annuler</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Nom du donneur</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>260</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Adresse du donneur</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>330</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Adresse Mail</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ajouter">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>490</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>410</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Numero du telephone</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_tlph">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>430</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Numéro de téléphone : 8 chiffres</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_supprimer">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>540</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_modifier">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>540</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_afficher">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>730</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher la liste</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pdf">
    <property name="geometry">
     <rect>
      <x>920</x>
      <y>730</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Exporter en PDF</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stat">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>730</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher statistiques</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_chercher">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>200</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Chercher par Nom ou ID</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>200</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Par défaut</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ordre ascendant ID</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ordre descendant ID</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Odre alphabétique des noms A-Z</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Odre alphabétique des noms Z-A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Région</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Adresse mail</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="widget_5" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>1121</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 0, 0)
</string>
    </property>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>361</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_12">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>261</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);

color: rgb(0, 0, 0);

font: 14pt &quot;MV Boli&quot;;</string>
     </property>
     <property name="text">
      <string>Gestion des donneurs </string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>1590</x>
      <y>20</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Déconnecter</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Desktop/logout.png</normaloff>../../Desktop/logout.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
